squire_core/src/state.rs

```diff
@@ -109,3 +164,49 @@ impl SessionStore for AppState {
         todo!()
     }
 }
+
+/*
```

Comment:
Unless we need to test private methods or fields, these tests should be integration tests. We'll need some kind of mechanism to pull a testing version of the DB or to connect to a remote CI/testing DB. For now, it might be best to gate these tests behind a feature flag, say `db-tests`. That way they can be accessed without needing to uncomment things.